pkg/perf/cpubench/cpubench_task.go

```diff
@@ -10,15 +10,20 @@ import (
 	"github.com/threefoldtech/zos/pkg/stubs"
 )
 
-const cpuBenchmarkTaskID = "CPUBenchmark"
-const cpuBenchmarkCronSchedule = "0 0 */6 * * *"
+const (
+	cpuBenchmarkTaskID       = "CPUBenchmark"
```

Comment:
since this will end up in json, i actually don't like CamelCase as json keys. It's better if we then use kebab-case or snake_case
I suggest `cpu-benchmark` it looks better in json output